blueprints/utils/resources/expected-results.json

```diff
@@ -67,6 +67,11 @@
   "/javascript/frameworks/polymer/polymer-router.found",
   "/javascript/frameworks/react/route-path.found",
   "/javascript/frameworks/react/index.html/search.found",
+  "/javascript/state-changing-requests/automatic-post.found",
```

Comment:
I would probably split the CL thematically, i.e., have one that deals with non-{GET,POST} cases, and one that has the event listener changes.